messaging/messaging_java/build.gradle

```diff
@@ -81,6 +81,14 @@ afterEvaluate {
              'https://github.com/google/flatbuffers.git',
              flatbuffersDir
       }
+      exec {
```

Comment:
This should probably be in a separate PR

Reply:
Added [here](https://github.com/firebase/firebase-cpp-sdk/pull/1444) - I'll keep this in this PR (so my build works) until that PR goes in.